Assets/MixedRealityToolkit.SDK/Features/UX/Scripts/PressableButtons/PressableButton.cs

```diff
@@ -48,8 +48,10 @@ public class PressableButton : MonoBehaviour, IMixedRealityTouchHandler
 
         [Header("Position markers")]
         [Tooltip("Used to mark where button movement begins. If null, it will be automatically generated.")]
+
         [SerializeField]
-        private Transform initialTransform;
+        [Tooltip("Ensures that the button can only be pushed from the front. Touching the button from the back or side is prevented.")]
+        private bool enforceFrontPush = true;
```

Comment:
Looks like this field is already declared above.

Reply:
Yeah we had a wonky merge conflict resolution.